src/shared/components/RadioButton/RadioButton.style.ts

```diff
@@ -89,15 +97,26 @@ export const StyledInput = styled.div<RadioButtonStyleProps>`
   }
```

Comment:
It's not your change, but I'm almost sure that we don't need this `&::before` above. Can you double-check this and remove it if it's unnecessary?

Reply:
That's correct! Fixed

Reply:
Reverting that, missed that it handles hover state